config/jobs/kubernetes-sigs/sig-windows/release-1.22-windows.yaml

```diff
@@ -0,0 +1,56 @@
+periodics:
+- annotations:
+    testgrid-alert-email: kubernetes-provider-azure@googlegroups.com
+    testgrid-dashboards: sig-release-1.22-informing, sig-windows-signal, sig-windows-1.22-release
+    testgrid-tab-name: aks-engine-windows-containerd-1.22
+  decorate: true
+  decoration_config:
+    timeout: 3h0m0s
+  extra_refs:
+  - base_ref: release-1.22
+    org: kubernetes
+    path_alias: k8s.io/kubernetes
+    repo: kubernetes
+  interval: 3h
+  labels:
+    preset-azure-cred: "true"
+    preset-azure-windows: "true"
+    preset-dind-enabled: "true"
+    preset-k8s-ssh: "true"
+    preset-service-account: "true"
+    preset-windows-private-registry-cred: "true"
+    preset-windows-repo-list-master: "true"
+  name: ci-kubernetes-e2e-aks-engine-azure-master-windows-containerd-1-22
+  spec:
+    containers:
+    - args:
+      - --test
+      - --up
+      - --down
+      - --build=quick
+      - --dump=$(ARTIFACTS)
+      - --deployment=aksengine
+      - --provider=skeleton
+      - --aksengine-admin-username=azureuser
+      - --aksengine-admin-password=AdminPassw0rd
+      - --aksengine-creds=$(AZURE_CREDENTIALS)
+      - --aksengine-download-url=https://aka.ms/aks-engine/aks-engine-k8s-e2e.tar.gz
+      - --aksengine-public-key=$(K8S_SSH_PUBLIC_KEY_PATH)
+      - --aksengine-private-key=$(K8S_SSH_PRIVATE_KEY_PATH)
+      - --aksengine-winZipBuildScript=$(WIN_BUILD)
+      - --aksengine-orchestratorRelease=1.22
+      - --aksengine-template-url=https://raw.githubusercontent.com/kubernetes-sigs/windows-testing/master/job-templates/kubernetes_containerd_master.json
+      - --aksengine-win-binaries
+      - --aksengine-deploy-custom-k8s
+      - --test_args=--node-os-distro=windows -docker-config-file=$(DOCKER_CONFIG_FILE)
+        --ginkgo.focus=\[Conformance\]|\[NodeConformance\]|\[sig-windows\]|\[sig-apps\].CronJob|\[sig-api-machinery\].ResourceQuota|\[sig-network\].EndpointSlice
+        --ginkgo.skip=\[LinuxOnly\]|\[Serial\]|\[Slow\]|\[alpha\]|GMSA|Guestbook.application.should.create.and.stop.a.working.application|device.plugin.for.Windows
+      - --ginkgo-parallel=4
+      command:
+      - runner.sh
+      - kubetest
+      image: gcr.io/k8s-testimages/kubekins-e2e:v20210721-2b77449-1.22
```

Comment:
I made sure we have the right kubekins for 1.22 from this fix: https://github.com/kubernetes/test-infra/pull/22990